recipes/dynare_preprocessor/meta.yaml

```diff
@@ -0,0 +1,39 @@
+package:
+  name: dynare-preprocessor
+  version: 0.0.1
+
+source:
+  url: https://git.dynare.org/Dynare/preprocessor/-/archive/6235e6907526d468e2c35f14c38db1e308698bc9/preprocessor.tar.gz
+  sha256: 06f76245ff5c6ac0811438cea6b817a321dc702c188b680e2a3191b54837c4ab
+
+build:
+  number: 0
+  skip: true  # [win or osx]
+
+requirements:
+  build:
+    - '{{ compiler("cxx") }}'
+    - meson
+    - bison
+    - flex
+  host:
+    - git-lfs
+    - gsl
+    - openlibm
+    - libboost-devel
+    - pkg-config
```

Comment:
Move pkg-config and git-lfs to the build section.

Reply:
I'm moved pkg-config and removed git-lfs as it appears it is not needed anymore (we download an archive instead of cloning)